VideoWeb/VideoWeb/ClientApp/src/app/services/events.service.ts

```diff
@@ -111,7 +111,7 @@ export class EventsService {
       'ReceiveMessage',
       (conferenceId: string, from: string, message: string, timestamp: Date, messageUuid: string) => {
         const date = new Date(timestamp);
-        const chat = new InstantMessage({conferenceId, id: messageUuid, from, message, timestamp: date});
+        const chat = new InstantMessage({conferenceId, id: messageUuid, from, message, timestamp: date });
```

Comment:
There are no changes here. Could you revert this file please.

Reply:
done